tests/testthat/test-plot_with_settings.R

```diff
@@ -233,6 +233,10 @@ testthat::test_that("type_download_srv download all types of files and change th
 })
 
 testthat::test_that("type_download_srv downloads a png file with different dimensions", {
+  if (!require("png", quietly = TRUE)) {
+    skip("png package not available")
+  }
```

Comment:
For this there is a dedicated testthat function - skip-if-not-installed

Reply:
good idea.